akka-projection-persistence-query/src/main/scala/akka/projection/provider/scaladsl/AkkaPersistenceQuerySourceProvider.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.provider.scaladsl
```

Comment:
Here using same pacakge as SourceProvider, `akka.projection.provider.scaladsl`. For lack of better one. 

Tried with `akka.projection.query`, but can create confusion as people may think it's about querying the read side. 

Since this an implementation of `SourceProvider`, I'm adding it to the same package.

Suggestions welcome.